src/Apps/Sell/Routes/ThankYouRoute.tsx

```diff
@@ -1,41 +1,57 @@
 import { SubmissionLayout } from "Apps/Sell/Components/SubmissionLayout"
+import { RouterLink } from "System/Components/RouterLink"
+import { AppContainer } from "Apps/Components/AppContainer"
 
-const FRAGMENT = graphql`
-  fragment ThankYouRoute_submission on ConsignmentSubmission {
-    externalId
-  }
-`
-interface ThankYouRouteProps {
-  submission: ThankYouRoute_submission$key
-  children: React.ReactNode
-}
+export const ThankYouRoute: React.FC = () => {
+  return (
+    <FullBleed>
+      <AppContainer>
+        <SubmissionLayout hideNavigation>
+          <Join separator={<Spacer y={4} />}>
+            <Join separator={<Spacer y={2} />}>
+              <Text variant="lg-display">
+                Thank you for submitting your artwork
+              </Text>
 
-export const ThankYouRoute: React.FC<ThankYouRouteProps>= (props) => {
-  const submission = useFragment(FRAGMENT, props.submission)
+              <Text variant="xs">
+                An Artsy Advisor will email you within 3-5 days to review your
+                submission and discuss next steps. In the meantime your
+                submission will appear in the feature, My Collection.
+              </Text>
 
-  return (
-    <SubmissionLayout hideNavigation>
-      <Text variant="lg-display">Thank you for submitting your artwork </Text>
-      <Spacer y={2}/>
-      <Text variant="xs">ID: {submission.externalId}</Text>
-      <Spacer y={2}/>
-      <Text variant="xs">
-        An Artsy Advisor will email you within 3-5 days to review your submission
-        and discuss next steps. In the meantime your submission will appear in the
-        feature, My Collection.
-      </Text>
-      <Spacer y={2}/>
-      <Message variant="info" title="What happens next?">
-        <Text variant="xs">
-          If your artwork is accepted, we will send you a sales offer and guide
-          you in selecting the best selling option. Additional information may be
-          requested.
-        </Text>
-      </Message>
-    </SubmissionLayout>
+              <Message variant="success" title="What happens next?">
+                <Text variant="xs">
+                  If your artwork is accepted, we will guide you in selecting
+                  the best selling option. Additional information may be
+                  requested.
+                </Text>
+              </Message>
+            </Join>
+
+            <Join separator={<Spacer y={2} />}>
+              <Button // @ts-ignore
+                as={RouterLink}
+                to="/sell2/submissions/new"
+                width="100%"
+              >
+                Submit Another Work
+              </Button>
+
+              <Button
+                // @ts-ignore
+                as={RouterLink}
+                to="/collector-profile/my-collection"
+                variant="secondaryBlack"
+                width="100%"
+              >
+                View Artwork in My Collections
+              </Button>
```

Comment:
This link doesn't work as expected -- instead of being navigated to My Collection when pressing the button, i see the url changing into `/collector-profile/my-collection` and then back to `/thank-you`. I think it might be because we moved this step in [this PR](https://github.com/artsy/force/pull/14041/files#diff-91221e8e1ff465fde08b9f4df582f30da99522b0efe825318ebb2594065d26f8R270). Do you know what has to be done for the link to work as expected?

Reply:
Hmm, I really don't know 🤔 Maybe this is something for #dev-help